superset/db_engine_specs/base.py

```diff
@@ -98,7 +104,7 @@ def _constructor(self):
 
 @compiles(TimestampExpression)
 def compile_timegrain_expression(
-    element: TimestampExpression, compiler: Compiled, **kw
+    element: TimestampExpression, compiler: Compiled, **kw: Any
```

Comment:
while you're here, maybe we should change this to `kwargs` from `kw` to match the rest of the codebase